pkg/plugin/query_test.go

```diff
@@ -363,7 +363,7 @@ func TestQuery_parseLegend(t *testing.T) {
 			"job": "vmstorage-maas",
 		},
 		expr: "sum(vm_http_request_total)",
-		want: "sum(vm_http_request_total)",
```

Comment:
If legend format auto, then prometheus datasource returns the query itself, so I think the tests are correct 

<img width="1287" height="887" alt="Image" src="https://github.com/user-attachments/assets/f5bb89c9-d4b3-4d97-8884-fdc499ad3589" />

Reply:
@dmitryk-dk prometheus datasource in with legent=auto returns all labels instead of query name

Reply:
If it has grouping like `by` then yes it shows labels, but if we do not use grouping, it shows the query itself
<img width="1164" height="931" alt="Screenshot 2025-08-13 at 10 24 07" src="https://github.com/user-attachments/assets/224c1b61-0892-489d-8dfa-6adedfb320bb" />

Reply:
I think we need to update the tests to show that the query uses grouping with `by` for better understanding, because labels can come with the grouping operator